src/hooks/useTaskStatuses.js

```diff
@@ -0,0 +1,29 @@
+import {useEffect, useState} from "react";
+
+const useTaskStatuses = (tasks) => {
+    const [todo, setTodo] = useState([]);
+    const [readyForDev, setReadyForDev] = useState([]);
+    const [inProgress, setInProgress] = useState([]);
+    const [done, setDone] = useState([]);
+
+    useEffect(() => {
+        const fTodo = tasks?.filter((task) => task.status === "todo") || [];
```

Comment:
1. create function getFilteredTasks in folder utilis 
2. func should have argument - status, tasks
3. return filtered array based on status 
4. call this func inside <Board/>

Reply:
done